documents/validator.go

```diff
@@ -178,13 +178,35 @@ func signaturesValidator(idService identity.ServiceDID) Validator {
 			return errors.New("atleast one signature expected")
 		}
 
+		authorFound := false
 		for _, sig := range signatures {
-			if erri := idService.ValidateSignature(&sig, sr); erri != nil {
+			tm, terr := utils.FromTimestamp(sig.Timestamp)
```

Comment:
Don't you want to remove the signature timestamp in this change as discussed? We don't need it and it's just misleading.

Reply:
I will remove, just that we didn't discuss what to do for collaborator signatures